src/compiler/parser.ts

```diff
@@ -2,15 +2,15 @@
 /// <reference path="utilities.ts"/>
 
 namespace ts {
-    const nodeConstructors = new Array<new (pos: number, end: number) => Node>(SyntaxKind.Count);
+    const nodeConstructors = new Array<new (pos: number, end: number, kind: number) => Node>(SyntaxKind.Count);

```

Comment:
No reason to have the `nodeConstructors` array anymore, there's just a single constructor now.